services/order.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Order, Ticket, User
+from django.db import transaction
+from datetime import datetime
+
+
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> Order:
+    with transaction.atomic():

```

Comment:
Using `transaction.atomic()` ensures that the order creation and the creation of associated tickets are done atomically, which is important for maintaining data consistency.